src/lib/agent/PolykeyAgent.ts

```diff
@@ -3,15 +3,20 @@ import fs from 'fs';
 import net from 'net';
 import path from 'path';
 import process from 'process';
-import { fork, ForkOptions } from 'child_process';
-import Polykey, { KeyManager } from '../Polykey';
 import Configstore from 'configstore';
+import { Duplex } from 'readable-stream';
 import PolykeyClient from './PolykeyClient';
 import { agent } from '../../../proto/js/Agent';
-import { Duplex } from 'readable-stream';
+import PeerInfo from '../../lib/peers/PeerInfo';
+import Polykey, { KeyManager } from '../Polykey';
+import { fork, ForkOptions } from 'child_process';
```

Comment:
Do note that there's no child process in nativescript https://stackoverflow.com/questions/41114647/nativescript-requirechild-process-gives-an-error-when-running-tns-run-ios

Not sure what this is used for atm. But we should be abstracting this if it is intended to be a worker process.

Reply:
yeah we will need to separate functionality like into some kind of interface that both the nodejs and nativescript modules for child processes. This is used for the daemon script btw.